app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts, foreign_key: :author_id
+  has_many :likes
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ Optional] Previous reviews mentioned about how user own all the created post under reference of author_id, but  hedid not mention that even comments and likes should belong to a user under reference of Aurthor_id as well.  Please like the way you did it on user-post relationship, do it to comment and like as well with a foreign key of author_id .

Reply:
Hello @ishimwezachee, the previous reviewer did mention this. But as I showed the diagram it is not using author_id for comments and likes. You can view the diagram on PR. Since my tables are created with user_id not author_id I think it is not easy to do these changes now because I have to change everything including my database. The requirement says to follow ERD and that is what I did.